### PR TITLE
First pass at the documentation for the new datastore.

### PR DESCRIPTION
**Pull request by fmizzell**
Connects #2569

### CODE REVIEW

File: docs/components/datastore.rst [hunk 1]
```diff
@@ -1,169 +1,79 @@
-DKAN Datastore
+Datastore
 ===============
```
Comment (janette) [inline docs/components/datastore.rst]: This opening bit reads a bit rough, can we start with how resources work, how the datastore enhances usability, and why it is good to take advantage of the option.

```
When you create a dataset with resources, DKAN is reading the data directly from the resource file or API.

The Datastore component provides an option for you to parse CSV files and save the data into database tables. This allows users to query the data through a public API.

So by adding your CSV resources to the datastore, you are getting the fullest functionality possible out of your datasets.
```

File: docs/components/datastore.rst [hunk 2]
```diff
@@ -1,169 +1,79 @@
...
--------------------------------
-**To create a datastore from a local file:**
+Configure the Manager
+^^^^^^^^^^^^^^^
 
-.. code-block:: php
+There are many ways to accomplish data querability. DKAN out of the box provides two ways to manage the process. The default manager is called **Simple Import**.
 
-  drush dsc (path-to-local-file)
+The first step on using the datastore is to configure which manager you would like to use for the resource.
```
Comment (janette) [inline docs/components/datastore.rst]: This first step to using... ? for using...?

File: docs/components/datastore.rst [hunk 3]
```diff
@@ -1,169 +1,79 @@
...
 
-**To update a datastore from a local file:**
+The **Simple Import** Manager will then need some information regarding the CSV format for proper parsing.
 
-.. code-block:: php
+- **delimiter**: the character that encloses the fields in your CSV file.
+- **escape**: the character used to escape other characters in your CSV file.
 
-  drush dsu (datastore-id) (path-to-local-file)
+Enter the character that are used as delimeter, quote and escape characters in your file, then use the "Import" button at the bottom of the page to import the data from your file or API into the datastore.
```
Comment (janette) [inline docs/components/datastore.rst]: Enter the character used for each function (delimeter, quote and escape) in your file, then click the "Import" button...

**Comment by janette** [comment]
datastore

**Comment by acouch** [comment]
It should also show if the table has been created yet or not. That is pretty key in debugging. Is that what it means by ``Unitialized``? Can we think of a clearer way of stating that or help text?

**Comment by kimwdavidson** [comment]
Pending conversation around storage state

**Comment by janette** [comment]
@fmizzell RE: 2615 test failure

The csv and pdf files return “application/xml” for type
```
use dkanDataset\getRemoteFileInfo;

include_once 'profiles/dkan/modules/dkan/dkan_dataset/includes/getRemoteFileInfo.php';

//$url ='https://chronicdata.cdc.gov/views/itia-u6fu/rows.csv?accessType=DOWNLOAD';
//$url = 'https://data.chhs.ca.gov/dataset/596b5eed-31de-4fd8-a645-249f3f9b19c4/resource/57da6c9a-41a7-44b0-ab8d-815ff2cd5913/download/cscpopendata.csv';
$url = 'https://data.chhs.ca.gov/dataset/cdfbe77b-abdc-4e2f-b1da-db47b2064b74/resource/e2127f3a-e712-412a-b931-ea151c053c03/download/2015-executive-summary.pdf';

$file_info = new GetRemoteFileInfo($url, 'test');

var_dump($file_info->getType());
var_dump($file_info->getName());
var_dump($file_info->getEffectiveUrl());

```

**Pull request merged**